src/Cmfd.cpp

```diff
@@ -139,7 +107,7 @@ void Cmfd::setNumY(int num_y) {
 
 /**
  * @brief Get the number of Mesh cells in a row.
- * @return number of Mesh cells in a row
+ * @return The number of Mesh cells in a row
```

Comment:
Many of these changes to the Oxygen comments are to address #117. I agree that having the first word capitalized is a good choice. I tried to update all the comments to consistently have the first word capitalized.